react-component/asu-react-isearch-view/.yarnrc

```diff
@@ -0,0 +1 @@
+registry "https://registry.web.asu.edu"
```

Comment:
Maybe the source of your yarn issues? Not hurting this PR, so I'm approving. But you may be able to remove this and then yarn would use your own .npmrc.